datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxInternalClusterRequest.java

```diff
@@ -1,12 +1,9 @@
 package com.sequenceiq.sdx.api.model;
 
-import javax.validation.Valid;
-
 import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.request.StackV4Request;
 
 public class SdxInternalClusterRequest extends SdxClusterRequest {
 
-    @Valid
```

Comment:
Is it just a quick fix? It is strange that we are removing the validation.

Reply:
i am not sure, in case of sdx internal, cloudbreak validates stackrequest anyway, but if we want to validate in earlier, we can do it, but we have to make everyone to put the name into the input json, and that will duplicate data on api, so maybe this worth a discussion

Reply:
I put up the validation a few days back, it is fairly safe to remove it. I did not calculate with the consequences that this object of the request is used in such a relaxed way.